app/org/catalyst/slackservice/db/RedisDbHandler.java

```diff
@@ -102,6 +103,21 @@ public Bot getBotInfo(String teamId) {
         return bot;
     }
 
+    @Override
+    public void deleteTokens(String teamId, String[] tokens) {
+        if (teamId == null || tokens == null || tokens.length == 0) {
+            logger.error("unable to delete tokens for team {}. token null? {}", teamId, tokens == null);
+            return;
+        }
+
+        String[] newTokens = Arrays.stream(tokens).map(token -> format(teamId, token)).toArray(String[]::new);
+
+        try (Jedis jedis = _jedisPool.getResource()) {
+            logger.debug("deleting tokens {}", Arrays.asList(newTokens));
+            jedis.hdel( USER_TOKENS, newTokens);
+        }
```

Comment:
Should this try have a corresponding catch?

Reply:
It's not necessary, but we could add one in for debugging / logging purposes. :) thanks!